app/src/main/java/co/develhope/meteoapp/data/DataSet.kt

```diff
@@ -0,0 +1,18 @@
+package co.develhope.meteoapp.data
+
+import co.develhope.meteoapp.model.RecentSearch
+
+class DataSet {
```

Comment:
Cambiaerei il nome in qualcosa di più parlante, altrimenti rischiamo di dover aprire il file ogni volta che cerchiamo un'informazione